core/common/src/main/java/alluxio/underfs/UnderFileSystemRegistry.java

```diff
@@ -114,6 +114,7 @@ public static UnderFileSystem create(String path, Object ufsConf) {
         return factory.create(path, ufsConf);
       } catch (Exception e) {
         errors.add(e);
+        LOG.warn("Failed to create ufs", e);
```

Comment:
The purpose of logging the exception here is to make the stack trace available. It would be strange to include stack traces in the detail message for the thrown `IllegalArgumentException`.